wordpress-mobile/WordPress-Android
# #title getting cut in settings screen fixed

# Title cut fix for NotificationsSettingsActivity and ReaderSubsActivity
### fix for #17325 

The title of screens was getting cut for some languages for the following screens.

1. Head to Reader and tap the "gear" icon
2. Head to Notifications and tap the "gear" icon.

How I fixed this issue.

I checked the internal implementation of MaterialToolbar and learned that by default it supports one line title and if it exceeds the one-line title gets cut. So instead of using the default title, I have created Textveiw inside the MaterialToolbar tag in XML and given the style property to make it as it was before. I've attached screenshots for you to look over.

Testing instructions:
  Go to your device's settings and change the language to Deutsch or German and check it won't get cut now.

![Screenshot_20230102_144707](https://user-images.githubusercontent.com/79354589/210213803-10943986-698a-41f8-8c94-b7c24144222b.png)
![Screenshot_20230102_144743](https://user-images.githubusercontent.com/79354589/210213817-8b111198-7e0a-48c4-aa2d-736ccfa40ba4.png)
